src/Libraries/RevitServices/Transactions/TransactionManager.cs

```diff
@@ -199,19 +199,19 @@ public class AutomaticTransactionStrategy : ITransactionStrategy
     {
         public TransactionHandle EnsureInTransaction(TransactionWrapper wrapper, Document document)
         {
-            TransactionManager.Log("EnsureInTransaction - AUTO STRAT: Starting new Transaction");
+            TransactionManager.Log("EnsureInTransaction - AUTO START: Starting new Transaction");
```

Comment:
@Randy-Ma Not sure. I don't know that this is a user facing message. If it is, I'll localize it.